src/connection.go

```diff
@@ -85,19 +97,26 @@ func (r redisConn) GetConfig() (map[string]string, error) {
 	return redis.StringMap(r.c.Receive())
 }
 
+// RenameCommands will populate internal renamedCommands mapping
+func (r redisConn) RenameCommands(renamedCommands map[string]string) {
+	if renamedCommands != nil {
+		r.renamedCommands = renamedCommands
+	}
```

Comment:
I would remove this `if`, setting a `nil` map is fine to remove all overrides.

https://play.golang.org/p/AWzrVAYxfkq